run_unittests.py

```diff
@@ -2015,6 +2015,10 @@ def test_libdir_must_be_inside_prefix(self):
         else:
             self.assertRaises(subprocess.CalledProcessError, self.setconf, '-Dlibdir=/opt', False)
 
+    def test_string_operations(self):
+      testdir = os.path.join(self.common_test_dir, '38 string operations')
+      self.init(testdir)
```

Comment:
you don't need to write unit test here, everything in common/ will be configured/build by default.

Reply:
btw, you can just append your tests to `common/38 string operations` where we already test other string methods. It's faster on the CI, it saves meson invocations on hundred small tests.

Reply:
what is the command to just run this unit test, I was unable to find it.

Reply:
I personally have a `~/.local/bin/meson` being a symlink to `meson.py` from source. Then I can just use meson command in test dirs. Dunno if there are better ways.

Reply:
`./meson.py ./run_unit_tests.py test_string_operations`

Reply:
yes i removed it at the end ... and thx @xavierclaessens I was dumb as I can run it directly in the folder ... :stuck_out_tongue: